repository/kafka/docs/latest/configuration.md

```diff
@@ -44,27 +44,47 @@ Deploying without persistent storage isn't recommended for production usage.
 
 By default, the Kafka cluster will have the JMX Exporter enabled. You can check more information around how KUDO Kafka exposes metrics in [monitoring](./monitoring.md).
 
-##### Zookeeper PATH
+##### Zookeeper Configuration
 
-By default, the Kafka cluster will try to use a zookeeper path same as specified in `--instance` flag.
-You can get the instance name by `kubectl get instances` 
+KUDO Kafka requires a running ZooKeeper ensemble to perform its own internal accounting and persist cluster topology. And connectivity to the Zookeeper.
```

Comment:
I personally like docs to be written in simple language so I would not use the word ensemble as I had to google it :)

Reply:
it comes from DC/OS docs

Reply:
using word `cluster` now